tfhe/src/core_crypto/commons/dispersion.rs

```diff
@@ -57,12 +59,12 @@ fn log2_modulus_to_modulus(log2_modulus: u32) -> f64 {
 /// assert_eq!(params.get_log_standard_dev().0, -25.);
 /// assert_eq!(params.get_variance().0, 2_f64.powf(-25.).powi(2));
 /// assert_eq!(
-///     params.get_modular_standard_dev(32).value,
+///     params.get_modular_standard_dev(32.0).value,
```

Comment:
this has to be wrong as you expect a modulus

Reply:
Done